connector/servicegraphconnector/README.md

```diff
@@ -120,7 +120,7 @@ datasources:
 
 The following settings are required:
 
-- `metrics_exporter`: the name of the exporter that this processor will write metrics to. This exporter **must** be present in a pipeline.
+- `metrics_exporter`: the name of the exporter that this connector will write metrics to. This exporter **must** be present in a pipeline.
```

Comment:
Is this still relevant since this is now a connector? I managed to get the connector working without using this field.